### PR TITLE
Add `fromHtml` example to readme.md

### PR DESCRIPTION
**Pull request by altano**
### Initial checklist

* [x] I read the support docs 
* [x] I read the contributing guide 
* [x] I agree to follow the code of conduct 
* [x] I searched issues and couldn’t find anything (or linked relevant results below) 
* [x] If applicable, I’ve added docs and tests

### Description of changes

The `fromHtml` utility is more convenient to use than hand-crafting the hast with `h`. Add an example that demonstrates how to use it in the readme

### CODE REVIEW

**Comment by altano** [comment]
Ideally this would just be abstracted away by `rehype-autolink-headings` (have `content` continue to accept hast but make a `htmlContent` and `svgContent` option to go alongside it that accepts content as a string?) but I figured we could start with a readme tweak.

**Comment by altano** [comment]
The unsuccessful checks do not appear related to my readme change.

File: readme.md [hunk 1]
```diff
@@ -173,6 +174,8 @@ By default, a `<span class="icon icon-link"></span>` is used.
 When a function is given, it’s called with the current heading (`Element`) and
 should return one or more nodes.
 
+The hast can be hand-written from scratch, built using the `h` builder from [`hastscript`](https://www.npmjs.com/package/hastscript), or compiled from a string using the `fromHtml` builder from [`hast-util-from-html`](https://www.npmjs.com/package/hast-util-from-html).
+
```
Comment (wooorm) [inline readme.md:178]:
```suggestion
```

I recommend removing this here. Explaining all possible ways how to generate hast will take a lot of docs. The docs here are about how this option works, having too much there makes it harder to figure that out.

File: readme.md [hunk 2]
```diff
@@ -272,6 +275,39 @@ Yields:
...
 
+### Example: content from `fromHtml` builder
+
+The following example passes `content` as a function, to generate an accessible
+description of each link.
+
+```js
+import {rehype} from 'rehype'
+import rehypeAutolinkHeadings from 'rehype-autolink-headings'
+import {fromHtml} from "hast-util-from-html"
```
Comment (wooorm) [inline readme.md:286]:
```suggestion
import {fromHtmlIsomorphic} from 'hast-util-from-html-isomorphic'
```

File: readme.md [hunk 3]
```diff
@@ -272,6 +275,39 @@ Yields:
...
+main()
+
+async function main() {
+  const file = await rehype()
+    .data('settings', {fragment: true})
+    .use(rehypeAutolinkHeadings, {
+      content: fromHtml(
+        `<svg height="10" width="10"><circle cx="5" cy="5" r="5" fill="black" /></svg>`,
+        {space: "svg", fragment: true}
+      ),
```
Comment (wooorm) [inline readme.md:297]:
```suggestion
      content: fromHtmlIsomorphic(
        '<svg height="10" width="10"><circle cx="5" cy="5" r="5" fill="black" /></svg>',
        {fragment: true}
      ).children
```

1. If you are going to include an HTML parser, `fromHtmlIsomorphic` is a better idea
2. Match code style
3. Use `children` from the root, so as not to include a `root`, which shouldn’t be done, this still allows for the theoretical case where a doctype is parsed by `fromHtml`, so the types don’t match, but that practically does not occur

**Review by wooorm — COMMENTED** [review]
I’m not opposed to including an example on how to do this. I don‘t think it’s a good idea to require an HTML parser in plugins though.

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thank you! :)

**Pull request merged**